vitess.io/_layouts/base.liquid

```diff
@@ -12,9 +12,6 @@
     <meta name="og:title" content="{{ site.title }} / {{page.title}}"/>
     <meta name="og:image" content="{{ site.baseurl }}/images/{{ site.logo }}"/>
     <meta name="og:description" content="{{ site.description }}"/>
-
-    <!-- GA -->
-    {% include analytics.liquid %}
```

Comment:
I moved this up to the `<head>` because the Google Analytics docs said to put it there. Is there a reason you want it in `<body>`?